remote_connection.go

```diff
@@ -0,0 +1,83 @@
+package influxdb
+
+import (
+	"context"
+
+	"github.com/influxdata/influxdb/v2/kit/platform"
+)
+
+type RemoteConnectionService interface {
+	// ListRemoteConnections returns all info about registered remote InfluxDB instances matching a filter.
+	ListRemoteConnections(context.Context, RemoteConnectionListFilter) (*RemoteConnections, error)
+
+	// CreateRemoteConnection registers a new remote InfluxDB instance.
+	// Before persisting the new connection info, the service verifies that the remote instance can be reached.
```

Comment:
Based on the API changes in https://github.com/influxdata/openapi/pull/180, is this still how it's going to work? I was understanding that it would be possible to create the remote without validation via the query parameter.